services/director-v2/src/simcore_service_director_v2/core/settings.py

```diff
@@ -155,7 +155,7 @@ class DynamicSidecarSettings(BaseCustomSettings):
     )
 
     DYNAMIC_SIDECAR_LOG_LEVEL: str = Field(
-        "WARNING",
+        "INFO",
```

Comment:
so I think this is how I would suggest proceeding:
- let this to warning
- just change the INFO to WARNING directly in the deployment repositories

Reply:
OK, let's do it like you suggest. I will think the devops notes are sufficient. for this